docs/src/lib/MDX_COMPONENTS.js

```diff
@@ -72,21 +72,21 @@ const cascaraComponents = {
   AreaPlaceholder: (props) => <AreaPlaceholder {...props} />,
   Boundaries: (props) => <Boundaries {...props} />,
   Button: (props) => <Button {...props} />,
-  ChatMessages: (props) => (
+  Chat: (props) => (
     <ChatProvider>
-      <ChatMessages {...props} />
+      <Chat {...props} />
```

Comment:
@lordpixel is this correct? It would seem that `ChatMessages` is quite different from `Chat` or not?

Reply:
Yes @odiseo42, That was the original name for the component, we added Messages suffix.